### ISSUE TITLE
Compatibility with Ubuntu

### ISSUE THREAD

**Issue by midarq** [e-kotov/writefull-for-sharelatex#1]
On Ubuntu, the script fails to find the correct folders.

**Comment by midarq** [comment]
Actually, it was a simple fix, here is the code:
```
7c8
< CHROME_BASE_DIR="${USER_HOME}/.config/google-chrome"
---
> CHROME_BASE_DIR="${USER_HOME}/Library/Application Support/Google/Chrome"
16c17
< for PROFILE in "${CHROME_BASE_DIR}/Default" "${CHROME_BASE_DIR}"/Profile*; do
---
> for PROFILE in "${CHROME_BASE_DIR}"/Profile*; do
64,65c65,66
< sed -i "s/overleaf\.com/${DOMAIN}/g" "$TMP_MANIFEST"
< sed -i "s/\"Writefull for Overleaf\"/\"Writefull for ${DOMAIN}\"/g" "$TMP_MANIFEST"
---
> sed -i '' "s/overleaf\.com/${DOMAIN}/g" "$TMP_MANIFEST"
> sed -i '' "s/\"Writefull for Overleaf\"/\"Writefull for ${DOMAIN}\"/g" "$TMP_MANIFEST"
```

**Comment by e-kotov** [comment]
@midarq hi! thank you for the interest, I'm glad that someone found this useful. Thank you for the suggested fix too. I will have time to test it next week when I get my hands on a Linux desktop with GUI. Do you want to submit a pull request?

**Comment by e-kotov** [comment]
@midarq hi, thanks again for the bug report and suggested fixes. I did not consider Linux at all in my initial script. Now I added support for Linux for Google Chrome, Chromium, and Chromium installed using snap (as the user data folders are slightly different for all of these).

Feel free to test:
```sh
git clone -b linux-fix https://github.com/e-kotov/writefull-for-sharelatex.git
cd writefull-for-sharelatex
chmod +x ./writefull_patch.sh
./writefull_patch.sh
```

I tested in a VM with `Chromium` installed from `snap` and it worked very well.

I will merge the fix #2 in a few days into the main branch.
